docs/components/dartpad/banners/regular/main.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Demo',
+      debugShowCheckedModeBanner: false,
+      home: MyHomePage(),
+      theme: ThemeData(
+        colorScheme: ColorScheme.light(),
+        primaryColor: ColorScheme.light().primary,
+        primaryColorDark: ColorScheme.light().primary,
```

Comment:
This makes the banner purple? If so LGTM.

Reply:
Yes it does